tools/generate-gemm-test.py

```diff
@@ -66,6 +66,32 @@ def split_ukernel_name(name):
     mr_packed = mr // int(mr_packed.group(1))
   else:
     mr_packed = mr
+  if "sme" in target_name:
```

Comment:
I don't think this should be duplicating the sme2 version below. Aren't they identical except for the appearance of `sme` vs. `sme2`?

Reply:
@dsharlet, we will fix it in a follow-up PR.